repose-aggregator/components/services/opentracing-service/opentracing-service-api/src/main/resources/META-INF/schema/examples/opentracing.cfg.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<opentracing xmlns="http://docs.openrepose.org/repose/opentracing/v1.0"
+             name="test-repose"
+             tracer="jaeger"
+             collector-endpoint="http://localhost:8081">
+    <jaeger-sampling-config sample-type="rate-limited">
```

Comment:
I have feedback on this config, but i want to give it more consideration before i make recommendations.